corehq/apps/users/views/mobile/users.py

```diff
@@ -743,6 +748,9 @@ def post(self, request, *args, **kwargs):
             if 'location_id' in request.GET:
                 user.set_location(loc)
 
+            if phone_number:
+                initiate_sms_verification_workflow(user, phone_number)
```

Comment:
No this shouldn't be slow, but you may want to inspect the return value here. For example, if this returns `VERIFICATION__ALREADY_IN_USE`, then it means that another contact has already verified that phone number.  Not sure if that matters here or not but just fyi